try:
    print a 
except Exception:
    print str(Exception)

print 'over'